### PR TITLE
Filteration and some suggested changes

### PR DESCRIPTION
**Pull request by aghilodessey**
-->Made the Root URL as project List
-->Ability to search the ProjectCode
-->Changed date format to Ist
-->When Clicked on Project Code..It will get navigated to logs of that specific project..And that projectcode will be displayed on project filteration
-->Ability to filter Based on Logs and Log Id
-->in "/logs" URL we can filter based on Project code also 
ph1
ph2

### CODE REVIEW

File: client/src/pages/logs-screen/LogTable.tsx [hunk 1]
```diff
@@ -28,36 +31,40 @@ interface LogItemInf {
   logType: string;
   created_at: string;
   metaTimestamp?: string; // @Todo will rename it once added to backend
```
Comment (nihiannd) [inline client/src/pages/logs-screen/LogTable.tsx]: No need of this line, Full string data come under `meta`

File: client/src/pages/logs-screen/LogTable.tsx [hunk 2]
```diff
@@ -28,36 +31,40 @@ interface LogItemInf {
...
-  const filter: FilterInf =
-    selectedLogType !== ""
-      ? {
-          logType: { IN: [selectedLogType] },
-        }
-      : {};
+  const filter: FilterInf = {
+    ...(selectedLogType
+      ? { logType: { IN: [selectedLogType] } }
+      : { logType: { IN: ["ERROR", "WARNING", "INFO"] } }),
```
Comment (nihiannd) [inline client/src/pages/logs-screen/LogTable.tsx]: No need providing all type, empty will get all data

File: client/src/pages/logs-screen/LogTable.tsx [hunk 3]
```diff
@@ -88,67 +95,72 @@ const LogTable: React.FC<LogTableProps> = ({ selectedLogType }) => {
...
+                  onClick={() => {
+                    navigate(`/logdetails/${item.logId}`);
+                  }}
+                >
+                  <Td>
+                    <Text color="teal" fontWeight="bold">
+                      {item.logId}
+                    </Text>
+                    <Text mt="1" mb="2">
+                      {" "}
```
Comment (nihiannd) [inline client/src/pages/logs-screen/LogTable.tsx]: `{" "}` Why this is given in many lines. How much difference is there with this

File: client/src/pages/logs-screen/LogTable.tsx [hunk 4]
```diff
@@ -88,67 +95,72 @@ const LogTable: React.FC<LogTableProps> = ({ selectedLogType }) => {
...
+                    >
+                      {item.logType}
+                    </Badge>
+                  </Td>
+
+                  <Td fontWeight="bold">{item.created_at}</Td>
+                  <Td>
+                    <Text mb="2" fontWeight="bold">
+                      {item.metaTimestamp ? item.metaTimestamp : "N/A"}
+                      {/* @Todo currently static, once added to Api it will be made dynamic */}
```
Comment (nihiannd) [inline client/src/pages/logs-screen/LogTable.tsx]: No need this

**Review by nihiannd — CHANGES_REQUESTED** [review]

File: client/src/pages/logs-screen/LogTable.tsx (hunk 2 above)
Comment (aghilodessey) [reply client/src/pages/logs-screen/LogTable.tsx]: Done

File: client/src/pages/logs-screen/LogTable.tsx (hunk 3 above)
Comment (aghilodessey) [reply client/src/pages/logs-screen/LogTable.tsx]: Removed it..It was Automatically Generating when i save the file

File: client/src/pages/logs-screen/LogTable.tsx (hunk 4 above)
Comment (aghilodessey) [reply client/src/pages/logs-screen/LogTable.tsx]: Okaay Done

File: client/src/pages/logs-screen/LogTable.tsx (hunk 1 above)
Comment (aghilodessey) [reply client/src/pages/logs-screen/LogTable.tsx]: Okaay done

**Review by nihiannd — APPROVED** [review]

**Pull request merged**